handsontable/src/editors/textEditor/textEditor.js

```diff
@@ -220,6 +225,8 @@ export class TextEditor extends BaseEditor {
     this.TEXTAREA.setAttribute('data-hot-input', ''); // Makes the element recognizable by Hot as its own component's element.
     this.TEXTAREA.tabIndex = -1;
 
+    this.TEXTAREA.setAttribute('aria-hidden', 'true');
```

Comment:
```suggestion
    this.TEXTAREA.setAttribute('data-hot-input', ''); // Makes the element recognizable by Hot as its own component's element.
    this.TEXTAREA.setAttribute('aria-hidden', 'true');
    this.TEXTAREA.tabIndex = -1;
```
WDYT about grouping that two lines that changes attributes of the element?